boost_src/sqlite3pp.cpp

```diff
@@ -483,7 +483,10 @@ namespace sqlite3pp
 
   transaction::transaction(database& db, bool fcommit, bool freserve) : db_(&db), fcommit_(fcommit)
   {
-    db_->execute(freserve ? "BEGIN IMMEDIATE" : "BEGIN");
+    //http://www.sqlite.org/rescode.html#busy
```

Comment:
Let's remove this comment line just to be consistent with other lines that has no links to sqlite docs.